src/ol/interaction/drawinteraction.js

```diff
@@ -424,8 +431,9 @@ ol.interaction.Draw.prototype.startDrawing_ = function(event) {
 ol.interaction.Draw.prototype.modifyDrawing_ = function(event) {
   var coordinate = event.coordinate;
   var geometry = this.sketchFeature_.getGeometry();
-  var coordinates, last;
-  if (this.mode_ === ol.interaction.DrawMode.POINT) {
+  var coordinates, last, sketchLineGeom;
+  if (this.mode_ === ol.interaction.DrawMode.POINT)
+  {
```

Comment:
Oops, sorry. My company's coding style is to put new lines before braces, so that's my habit. Thought I'd got them all, but looks like I missed one. I'll fix that shortly.